<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="GLWidget" name="myGLWidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_Shader"/>
    <addaction name="actionReload_Shader"/>
   </widget>
   <widget class="QMenu" name="menuDebug">
    <property name="title">
     <string>Debug</string>
    </property>
    <addaction name="actionToggle_Debug_Info"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuDebug"/>
  </widget>
  <action name="actionOpen_Shader">
   <property name="text">
    <string>Open Shader</string>
   </property>
  </action>
  <action name="actionReload_Shader">
   <property name="text">
    <string>Reload Shader</string>
   </property>
  </action>
  <action name="actionToggle_Debug_Info">
   <property name="text">
    <string>Toggle Debug Info</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GLWidget</class>
   <extends>QWidget</extends>
   <header>glwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
